tensorboard/webapp/metrics/store/metrics_reducers_test.ts

```diff
@@ -1208,6 +1208,23 @@ describe('metrics reducers', () => {
       expect(nextState.settings.cardMinWidth).toBe(400);
       expect(nextState.settingOverrides.cardMinWidth).toBeNull();
     });
+
+    it('updates hideEmptyCards', () => {
+      const prevState = buildMetricsState({
+        settings: buildMetricsSettingsState({
+          hideEmptyCards: false,
+        }),
+        settingOverrides: {},
+      });
+
+      const nextState = reducers(
+        prevState,
+        actions.metricsHideEmptyCardsChanged()
+      );
+
+      expect(nextState.settings.hideEmptyCards).toBe(false);
+      expect(nextState.settingOverrides.hideEmptyCards).toBe(true);
```

Comment:
Can you also test that another metricsHideEmptyCardsChanged action will toggle this back to false?